### PR TITLE
add regiodesics and openscenegraph modules

### PR DESCRIPTION
**Pull request by ppodhajski**

### CODE REVIEW

File: var/spack/repos/builtin/packages/openscenegraph/package.py [hunk 1]
```diff
@@ -12,15 +12,17 @@ class Openscenegraph(CMakePackage):
...
+    url      = "https://github.com/openscenegraph/OpenSceneGraph/archive/OpenSceneGraph-3.2.3.zip"
 
-    version('3.2.3', '02ffdad7744c747d8fad0d7babb58427')
-    version('3.1.5', '1c90b851b109849c985006486ef59822')
+    version('3.2.3', sha256='1e15f2c92e2e4a396f42fd4a8b64b31a65d2036b9fb5d1555a798a81598a533d')
+    version('3.1.5', sha256='1ca3f16ec023486686087c832e64e54da81cc12344549ba26ededfd9295572bc')
 
     variant('shared', default=True, description='Builds a shared version of the library')
 
+    patch('relwithdebinfo_postfix.patch')
```
Comment (pramodk) [inline var/spack/repos/builtin/packages/openscenegraph/package.py:22]: Is this patch applicable for all versions? Or, only latest?

File: var/spack/repos/builtin/packages/regiodesics/package.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+class Regiodesics(CMakePackage):
+
+    git = "ssh://bbpcode.epfl.ch/viz/Regiodesics"
+
+    version('0.1.0', tag='0.1.0', submodules=True)
+
+    depends_on('cmake@3.1:', type='build')
+    depends_on('ninja', type='build')
+    depends_on('bbptestdata')
```
Comment (pramodk) [inline var/spack/repos/builtin/packages/regiodesics/package.py:16]: Isn't this test dependency? (i.e. bbptestdata might be needed for tests?)

File: var/spack/repos/builtin/packages/regiodesics/package.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    depends_on('boost@1.65.0')
```
Comment (pramodk) [inline var/spack/repos/builtin/packages/regiodesics/package.py:17]: can we relax this? I assume this builds with other boost versions as well?

**Review by pramodk — COMMENTED** [review]
fee minnor comments

File: var/spack/repos/builtin/packages/regiodesics/package.py (hunk 3 above)
Comment (matz-e) [reply var/spack/repos/builtin/packages/regiodesics/package.py:17]: This should be revisited with the new deployment release.

**Review by matz-e — CHANGES_REQUESTED** [review]
The CI will be re-enabled next week, and provide boost 1.70.0 (IIRC). I think we should get by without pinning boost.

File: var/spack/repos/builtin/packages/regiodesics/package.py (hunk 3 above)
Comment (matz-e) [reply var/spack/repos/builtin/packages/regiodesics/package.py:17]: @ppodhajski do you absolutely have to have 1.65?

**Comment by pramodk** [comment]
@matz-e : can be closed I assume?

**Comment by matz-e** [comment]
Ah, @ppodhajski sorry, I didn't see this and duplicated the work…

**Pull request closed without merging**